platform/pulp/tasking/tasks.py

```diff
@@ -225,14 +230,18 @@ def apply_async(self, *args, **kwargs):
 
         :param group_id:    The id that identifies which group of tasks a task belongs to
         :type group_id:     uuid.UUID
+        :param args:          The positional arguments to pass on to the task.
```

Comment:
A few lines above L#223 - L#225 has queue as a parameter but it's not accepted as a named parameter. Please remove queue from this docstring.